### DISCUSSION TITLE
Getting User Info Discord OAuth2

### DISCUSSION THREAD

**Discussion by gaurishhs** [nuxt-community/auth-module#1641] [Support Requests]
# Problem
- So, I'm authorizing the user with guilds scope to know which guilds is he joined but i dont know how do i get the info? Or how to even get the access token? please help me get the servers the user is joined to

**Comment by Intevel** [comment]
You have to adjust the scopes to the ones given in the Discord API.

**Reply by gaurishhs** [reply]
i did add scopes, yet i dont get the data
